solutions/lamp-stack-vm-v1/configuration.metadata.json

```diff
@@ -140,6 +140,7 @@
                 "tooltip": "The expected maintenance overhead associated with managing this configuration."
             },
             "templateFileName": "azuredeploy-small2mid-noha.json",
+            "uiDefinitionFileName": "azuredeploy-minimal-uidefinition.json",
```

Comment:
if you're reusing the same form definition, consider removing the "minimal" from the name ? It gives the impression that there are config specific form definitions that we've not linked correctly.

Reply:
Renamed the form definition for both the LAMP and wordpress solutions to `uiformdefinition.json`